core/src/main/java/org/apache/iceberg/encryption/PlaintextEncryptionManager.java

```diff
@@ -18,26 +18,32 @@
  */
 package org.apache.iceberg.encryption;
 
-import java.nio.ByteBuffer;
 import org.apache.iceberg.io.InputFile;
 import org.apache.iceberg.io.OutputFile;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class PlaintextEncryptionManager implements EncryptionManager {
+  private static final EncryptionManager INSTANCE = new PlaintextEncryptionManager();
   private static final Logger LOG = LoggerFactory.getLogger(PlaintextEncryptionManager.class);
 
+  @Deprecated
```

Comment:
We typically add comments that state when the method will be removed and what to use instead. You can search for `@deprecated` in Javadoc to see examples.